docs/product-migration-guide/src/main/asciidoc/migration-assets-con.adoc

```diff
@@ -0,0 +1,26 @@
+
+Forms::
+Forms are migrated to {PRODUCT} {PRODUCT_VERSION} by a forms migration tool that is part of the {CENTRAL} migration tool. Migrated forms are managed in a new forms designer that currently does not support all features of the previous forms designer in {PRODUCT_OLD} 6.4. The migration tool and the forms designer will be incrementally improved in subsequent releases to better support currently deprecated features. Therefore, Red Hat recommends that you avoid deleting any migrated forms that lose functionality in migration because they may be supported in the future.
++
+--
+The following limitations and considerations apply to form migration:
+
+* Existing forms in {PRODUCT_OLD} 6.4 must be correct. If a conflict arises during migration, some forms might not be migrated.
+* Forms related to business processes (forms that end with a `taskform` suffix) no longer support fields with nested bindings (such as client/name) caused by data object variables on the process. Any forms with nested bindings will result in additional SubForms during migration.
+* Forms that are not related to business processes (forms that do not end with a `taskform` suffix) must have only one `DataHolder` of type `DataObject` or `ClassName`.
+* The new forms designer does not support different bindings for input and output of processes. The same binding must be assigned to the input and output of processes.
+* The new forms designer does not support dynamic functions, such as field calculations.
+--
+Business processes::
+Business processes migrated to {PRODUCT} {PRODUCT_VERSION} by the {CENTRAL} migration tool are managed in the same process designer as {PRODUCT_OLD} 6.4. Any new business processes created in {PRODUCT} {PRODUCT_VERSION} are managed by default in the new process designer. The new designer does not support all features of the previous process designer, but will be incrementally improved in subsequent releases to better support currently deprecated features. If you encounter features not supported in the new process designer as you create new processes, you can switch to the previous designer for more advanced capabilities until support is restored in future releases.
++
+For more information about creating and managing processes in {PRODUCT} {PRODUCT_VERSION}, see Getting started with business processes.
+//@link: Add link to Michele's processes doc.
+
+Pages (previously Dashboards)::
+Dashboards in {PRODUCT_OLD} 6.4 cannot be migrated to {PRODUCT} {PRODUCT_VERSION}. Any existing dashboards must be re-created as Pages in {PRODUCT} {PRODUCT_VERSION}. For information about creating and managing pages, see Building custom pages.
```

Comment:
as pages in... (cap consistency)

Reply:
Yeah, I was torn between our sentence-style requirements plus making it understandable, so I went with just the first-mention rule. But you're right.